packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.js

```diff
@@ -129,8 +129,8 @@ describe('ChangeEventPlugin', () => {
       <input type="radio" onChange={cb} />,
     );
     setUntrackedValue(input, true);
-    ReactTestUtils.SimulateNative.click(input);
-    ReactTestUtils.SimulateNative.click(input);
+    ReactTestUtils.SimulateNative.change(input);
+    ReactTestUtils.SimulateNative.change(input);
     expect(called).toBe(1);
   });
```

Comment:
Am I correct to assume this is breaking change? Is this a change we are comfortable sending out in a minor release?

Reply:
that's a good question, technically i'd be relying on a private implementation detail, but i can see it breaking tests for folks as well. I defer to what the team thing on this

Reply:
I am comfortable with this, but it should go out in a minor. I'd hate for tests to fail on a patch.